tools/distpartitioning/dist_lookup.py

```diff
@@ -113,8 +117,9 @@ def __init__(self, input_dir, ntype_names, id_map, rank, world_size):
         self.ntype_count = np.array(ntype_count, dtype=np.int64)
         self.rank = rank
         self.world_size = world_size
+        self.num_parts = num_parts
 
-    def get_partition_ids(self, global_nids):
+    def get_partition_ids(self, agg_global_nids):
```

Comment:
why prepending `agg_` to the variable name?

Reply:
With this change now, this function now supports sending multiple smaller messages instead of one large message. So I used 'agg_', in this case, 'agg_global_nids' to indicate 'aggregate global nids' which this function is processing. When sending each smaller message out, we slice agg_global_nids to get the appropriate message slice and then send it for processing. Similarly the response's to get_partitionids is aggregated (stored) in the agg_partition_ids numpy array. 

At the end of all processing, agg_partition_ids are returned to the. caller. 

If you disagree with this naming convention, please suggest an appropriate names for the variables and I will make the necessary code changes.